docs/graphql/core/guides/postgres/import-data.rst

```diff
@@ -0,0 +1,66 @@
+.. meta::
+   :description: import data from csv into postgres
+   :keywords: hasura, docs, postgres, import, data
+
+.. _postgres_import_data_from_csv:
+
+Import data from CSV into Postgres
+===================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+-------------
+
+You might have some existing data in a CSV file that you need to import into your PG database. The following
+guide will show how to do so.
+
+In this guide we will use the following CSV file as an example. 
```

Comment:
@marionschleifer can I go with `Let's assume the following CSV file is named ``profile.csv`` instead of `Let's assume the following CSV file named ``profile.csv``
🙂